### PR TITLE
[#487][#488] Fix cloud2edge post-install hook, fix tour instructions.

### PR DESCRIPTION
**Pull request by calohmn**
Includes these fixes:
- #488 Fix cloud2edge post-install hook.
- #487 Fix HTTP endpoint usage in cloud2edge tour.
Using new REGISTRY_BASE_URL and HTTP_ADAPTER_BASE_URL environment variables that will contain either the secure or insecure endpoint base URL depending on the chart configuration.
- Adapt cloud2edge NOTES.txt output to used config.
- Replace obsolete cloud2edge profileKafkaMessaging.yaml.
Kafka usage is now the default.

### CODE REVIEW

File: homepage/_packages/cloud2edge/tour.md [hunk 1]
```diff
@@ -63,27 +66,27 @@ The demo device's digital twin supports a temperature property which will be set
 by means of the following command:
 
 {% clipboard %}
-curl -i -u demo-device@org.eclipse.packages.c2e:demo-secret -H 'application/json' --data-binary '{
+curl -i -k -u demo-device@org.eclipse.packages.c2e:demo-secret -H 'application/json' --data-binary '{
```
Comment (calohmn) [inline homepage/_packages/cloud2edge/tour.md:69]: FYI: I've added the `-k` (i.e. `--insecure`) option here for now. Better would be to fetch the used CA cert beforehand (via `kubectl get secret`) and use that here instead. But it seems the CA cert is currently only set in a K8s secret if Kafka messaging is being used. That means a generic solution requires changes also to the Hono chart.

File: homepage/_packages/cloud2edge/tour.md [hunk 2]
```diff
@@ -35,9 +35,12 @@ Download the [setCloud2EdgeEnv script](../scripts/setCloud2EdgeEnv.sh) and use i
 to set environment variables for the Cloud2Edge package's service endpoints.
 
 {% clipboard %}
+curl https://www.eclipse.org/packages/packages/cloud2edge/scripts/setCloud2EdgeEnv.sh --output setCloud2EdgeEnv.sh
+chmod u+x setCloud2EdgeEnv.sh
+
 RELEASE=c2e
 NS=cloud2edge
-./setCloud2EdgeEnv.sh $RELEASE $NS
+eval "$(./setCloud2EdgeEnv.sh $RELEASE $NS)
```
[the remark this answers is not in the document]
Comment (calohmn) [reply homepage/_packages/cloud2edge/tour.md]: yes, missing `"` at the end. Fixed.

[the remark this answers is not in the document]
Comment (sophokles73) [reply homepage/_packages/cloud2edge/tour.md]: FMPOV there is no need for the quotes at all, or am I missing something?

**Review by sophokles73 — APPROVED** [review]
LGTM

**Pull request merged**